tests/unit/cluster/failover2.tcl

```diff
@@ -64,3 +64,36 @@ start_cluster 3 4 {tags {external:skip cluster} overrides {cluster-ping-interval
     }
 
 } ;# start_cluster
+
+
+start_cluster 7 3 {tags {external:skip cluster} overrides {cluster-ping-interval 1000 cluster-node-timeout 5000}} {
```

Comment:
Did the test fail without the fix?

Reply:
we have a random delay time so the test probably won't fail in most cases

if we do some trick around failover_auth_time like removing the `random() % 500`, we got a big chance the test will fail without the fix

Reply:
```
        server.cluster->failover_auth_time = mstime() +
                                             500 + /* Fixed delay of 500 milliseconds, let FAIL msg propagate. */
                                             random() % 500; /* Random delay between 0 and 500 milliseconds. */
```

We can add some debug configuration items to each 500 and random 500 to increase conflicts. I usually adjust them while debugging